### PR TITLE
Add r/w logs rules support

### PR DESCRIPTION
**Pull request by periklis**
The following PR implements the client API for read/write access to logs-based rules

Depends on:
* observatorium/api#385

### CODE REVIEW

**Review by saswatamcode — APPROVED** [review]
Thanks! This looks good to me! 🚀 

Some suggestions! Maybe in a future PR we can add an e2e test for this?

**Comment by periklis** [comment]
> Thanks! This looks good to me! rocket
> 
> Some suggestions! Maybe in a future PR we can add an e2e test for this?

I am already writing e2e tests but not finished yet. This will be part of this PR.

File: README.md [hunk 1]
```diff
@@ -284,8 +285,11 @@ Usage:
   obsctl logs get [command]
 
 Available Commands:
+  alerts      Get alerts of a tenant.
   labels      Get labels of a tenant.
   labelvalues Get label values of a tenant.
+  rules       Get rules of a tenant.
+  rules.raw   Get rules of a tenant.
```
[the remark this answers is not in the document]
Comment (saswatamcode) [reply README.md]: Ah missed it! Thanks!

**Review by saswatamcode — APPROVED** [review]
Thanks for awesome work! 🌟

**Review by saswatamcode — APPROVED** [review]

**Review by matej-g — APPROVED** [review]
Nice :+1:

**Pull request merged**